src/simtools/applications/derive_limits.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+
+r"""
+Derives the limits for energy, radial distance, and viewcone using the LimitCalculator.
+
+The limits are derived based on the event loss fraction specified by the user.
+
+Command line arguments
+----------------------
+hdf5_file (str, required)
+    Path to the HDF5 file containing the event data.
+loss_fraction (float, required)
+    Fraction of events to be lost.
+
+
+Example
+-------
+Derive limits for a given HDF5 file with a specified loss fraction.
+
+.. code-block:: console
+
+    simtools-derive-limits\\
+        --hdf5_file path/to/hdf5_file.hdf5 \\
+        --loss_fraction 0.1
```

Comment:
This is probably a rather unrealistic loss fraction or? Do you want to give a realistic value in the example?